Add docs for CircleCI, Gitlab, and Tekton (#243)
buildpacks/docs: content/docs/tools/circleci.md

+++
title="CircleCI"
+++

> [CircleCI][circleci] is a continuous integration and delivery platform. The CNB project maintains an integration, called an “orb”, 

Reviewer: ```suggestion
[CircleCI][circleci] is a continuous integration and delivery platform. The CNB project maintains an integration, called an [orb](https://circleci.com/orbs/), 
```

> which allows users to utilize [pack][pack] inside their pipelines.

Reviewer: ```suggestion
which allows users to run [pack][pack] commands inside their pipelines.
```

<!--more-->

## Use

> To use the CNB integration, you need to declare that you are using the `buildpacks/pack` "orb”, and then utilize

Reviewer: ```suggestion
To use the CNB integration, you need to declare that you are using the [`buildpacks/pack` orb](https://circleci.com/developer/orbs/orb/buildpacks/pack), and then use
```
